src/main/java/edu/ucsb/cs156/example/entities/MenuItemReview.java

```diff
@@ -0,0 +1,29 @@
+package edu.ucsb.cs156.example.entities;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "menuitemreview")
+public class MenuItemReview {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  private long itemId;
+  private String reviewerEmail;
+  private int stars;
+  private String comments;
```

Comment:
Maybe the order of Reviews and comments should be switched. In the documentation it seems that the dateReviewed column comes before comments.